polaris-react/src/components/InContextLearning/InContextLearning.scss

```diff
@@ -1,11 +1,17 @@
+@import '../../styles/common';
+
 .InContextLearning {
   position: relative;
-  width: 250px;
+  width: 380px;
```

Comment:
Would we want to define these are variables? (ie desktop-width or something)

Reply:
For a future iteration, sure, absolutely I think we should. There's so much fast change going on within these components at the moment though. I didn't want to enshrine those values to variables, have another PR leverage those, and then need to later undo those changes.